scripts/installation/install.sh

```diff
@@ -78,7 +78,91 @@ delete_crds() {
   kubectl delete -f build/crds --timeout=90s
 }
 
+create_kb(){
+cd ${SEDNA_ROOT}
+kubectl create namespace sedna
+kubectl label node/$KB_NODE_NAME sedna=kb --overwrite
+
+kubectl $action -f - <<EOF
+apiVersion: v1
+kind: Service
+metadata:
+  name: kb
+  namespace: sedna
+spec:
+  selector:
+    sedna: kb
+  type: NodePort
+  ports:
+    - protocol: TCP
+      port: 9020
+      targetPort: 9020
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kb
+  labels:
+    sedna: kb
+  namespace: sedna
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      sedna: kb
+  template:
+    metadata:
+      labels:
+        sedna: kb
+    spec:
+      nodeSelector:
+        # only schedule to master node
+        node-role.kubernetes.io/master: ""
```

Comment:
use [node affinity](https://kubernetes.io/zh/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity) DoesNotExist operator to schedule into non-edge nodes

Reply:
If the KB server is limited into a specified node currently, then use the node of GM